package.json

```diff
@@ -5,19 +5,20 @@
   "license": "BSD-3-Clause",
   "repository": "https://github.com/GPII/gpii-binder",
   "scripts": {
+    "pretest": "node_modules/rimraf/bin.js coverage instrumented",
     "test": "node node_modules/testem/testem.js ci --skip PhantomJS,Chromium,Safari"
   },
   "devDependencies": {
-    "eslint": "2.11.1",
+    "eslint": "3.2.2",
     "grunt": "1.0.1",
-    "grunt-eslint": "18.1.0",
-    "grunt-jsonlint": "1.0.4",
+    "fluid-grunt-eslint": "18.1.1",
```

Comment:
Linting doesn't function because the gruntfile wasn't updated for this dependency change